src/services/Functional/ErrorHandling.js

```diff
@@ -0,0 +1,10 @@
+import { useToast } from "vue-toastification";
+const toast = useToast();
+
+export default {
+  handleAPIErrors(error) {
+    // TODO - Add more types of errors here
+    if (error.message == "Network Error")
```

Comment:
can't we do it using response code? is the message always going to be this?

Reply:
https://github.com/axios/axios/issues/383